README.md

```diff
@@ -443,6 +445,12 @@ git tag --list 'v[0-9]*'
 
 A regex for skip processing the matched tags.
 
+#### ignore_tags
+
+A regex for ignore processing the matched tags.
+
+While `skip_tags` drop commits from the Changelog, `ignore_tags` include ignored commits into the next tag.
```

Comment:
Nit: I think we can make the word "Changelog" lowercase since it is written like that in other places in `README.md` (as "changelog")

Reply:
Fixed in 110e8b8fbd37f9215bfb0ab51e956320a7d3bfac.